### ISSUE TITLE
Problem: Auto Generated GUID collision in index.yaml

### ISSUE THREAD

**Issue by thebleucheese** [redcanaryco/atomic-red-team#2845]
## What did you do?

Attempted to parse and utilize contents of Atomic Red Team `index.yaml` file

## What did you expect to happen?

Successful parsing to code that requires auto_generated_guid to be a unique value.

## What happened instead?

There is an index collision and it failed due to the following tests:

```yaml
- name: Modify UseTPMKeyPIN Registry entry
      auto_generated_guid: 02d8b9f7-1a51-4011-8901-2d55cca667f9
      description: 'Allow startup key and PIN with TPM for Bitlocker tool

        '
      supported_platforms:
      - windows
      executor:
        command: 'reg add "HKLM\SOFTWARE\Policies\Microsoft\FVE" /v UseTPMKeyPIN /t
          REG_DWORD /d 2 /f

          '
        cleanup_command: 'reg delete "HKLM\SOFTWARE\Policies\Microsoft\FVE" /v UseTPMKeyPIN
          /f

          '
        name: command_prompt
        elevation_required: true
 - name: Modify EnableNonTPM Registry entry
      auto_generated_guid: 02d8b9f7-1a51-4011-8901-2d55cca667f9
```

## Your Environment

N/A - source code interacting with yaml file

**Comment by cyberbuff** [comment]
Hey @thebleucheese Thanks for reporting the issue. This should be fixed in #2837

**Comment by cyberbuff** [comment]
Fixed in #2837
